Makefile

```diff
@@ -14,8 +14,6 @@
 
 SHELL    := /bin/bash
 WEBROOT  := frontends/web
-GOPATH   ?= $(HOME)/go
```

Comment:
GOPATH can be any directory on your system. In Unix examples, we will set it to $HOME/go (the default since Go 1.8).